lib/transports/polling-xhr.js

```diff
@@ -401,7 +401,8 @@ if (typeof document !== 'undefined') {
   if (typeof attachEvent === 'function') {
     attachEvent('onunload', unloadHandler);
   } else if (typeof addEventListener === 'function') {
-    addEventListener('beforeunload', unloadHandler, false);
+    var terminationEvent = 'onpagehide' in self ? 'pagehide' : 'unload';
```

Comment:
@SQReder Could you please explain what motivates the change from `beforeunload` to `unload`?